.github/workflows/goreleaser.yml

```diff
@@ -59,10 +59,8 @@ jobs:
       run: |
         chartVersion=$(echo ${{ github.ref_name }} | cut -c 2-)
 
-        make core-chart IMAGE_TAG=${{chartVersion}}
-
-        sed -i "s/^KUBESTELLAR_VERSION:.*$/KUBESTELLAR_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
-        sed -i "s/^TRANSPORT_VERSION:.*$/TRANSPORT_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
+        sed -i "s/^.*KUBESTELLAR_VERSION:.*$/KUBESTELLAR_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
+        sed -i "s/^.*TRANSPORT_VERSION:.*$/TRANSPORT_VERSION: \"${chartVersion}\"/g" ${{ env.CORE_CHART_PATH }}/values.yaml
```

Comment:
The following is what I see in values.yaml for that now. It looks to me like it needs no adjustment when making a release.

```yaml
# TRANSPORT_VERSION: e.g., "0.24.0"; defaults to KUBESTELLAR_VERSION value, if not defined
```